backend/middlewares/validators/lessonValidators.ts

```diff
@@ -0,0 +1,40 @@
+import { Request, Response, NextFunction } from "express";
+import {
+  getApiValidationError,
+  validateArray,
+  validatePrimitive,
+} from "./util";
+import { getErrorMessage } from "../../utilities/errorUtils";
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable-next-line import/prefer-default-export */
+export const courseRequestDtoValidator = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  let body;
+  try {
+    body = JSON.parse(req.body.body);
+  } catch (e: unknown) {
+    return res.status(400).send(getErrorMessage(e));
+  }
+  if (!validatePrimitive(body.course, "string")) {
+    return res.status(400).send(getApiValidationError("course", "string"));
+  }
+  if (!validatePrimitive(body.title, "string")) {
+    return res.status(400).send(getApiValidationError("title", "string"));
+  }
+  if (!validatePrimitive(body.description, "string")) {
+    return res.status(400).send(getApiValidationError("description", "string"));
+  }
+  if (!validatePrimitive(body.image, "string")) {
+    return res.status(400).send(getApiValidationError("image", "string"));
+  }
+  if (!validateArray(body.content, "string")) {
```

Comment:
Replacing content with list of objects will also affect the validator, we can either write our own validator for the type or just not validate it, up to you.